components/coding/profileV2/achievementcomponenets/CodingBadgeUnit.tsx

```diff
@@ -0,0 +1,68 @@
+import React from "react";
```

Comment:
let's use snakeCase and correct the component spelling for the folder name

I'd probably also put the main AchievementComponent file in here if we're going to have a separate folder for this set of components